examples/guestbook/README.md

```diff
@@ -67,7 +67,82 @@ The web front end interacts with the redis master via javascript redis API calls
 
 This example requires a running Kubernetes cluster.  See the [Getting Started guides](../../docs/getting-started-guides/) for how to get started. As noted above, if you have a Google Container Engine cluster set up, go [here](https://cloud.google.com/container-engine/docs/tutorials/guestbook) instead.
 
-### Step One: Start up the redis master
+### Step One: Fire up the redis master service
```

Comment:
Before continuing to the gory details, we should also recommend:

* the quick walkthrough: https://github.com/kubernetes/kubernetes/tree/master/docs/user-guide#quick-walkthrough
* the thorough walkthough: https://github.com/kubernetes/kubernetes/tree/master/docs/user-guide#thorough-walkthrough
* the concept guide: https://github.com/kubernetes/kubernetes/tree/master/docs/user-guide#concept-guide